charts/nutanix-csi-storage/values.yaml

```diff
@@ -19,28 +19,32 @@ openshift:
   workerIscsiConfig: false
 
 # kubeletDir allows overriding the host location of kubelet's internal state.
+# for NKE cluster set it to "/var/nutanix/var/lib/kubelet"
 kubeletDir: "/var/lib/kubelet"
 
 # Global Settings for all pods
 
+# csiSecretDir is the directory where PC (Prism Central) secret is mounted.
+csiSecretDir: "/var/run/ntnx-secret-dir"
+
+# category configmap name and namespace
+csiCategoryConfigmapName: ntnx-cluster-configmap
+csiCategoryConfigmapNamespace: ntnx-system
```

Comment:
@subodh01 
why there is a static namespace here ?
any scenario where the ntnx-cluster-configmap would not be in the same namespace than the csi controller ?

Reply:
will be fixed in beta2